hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/test/java/org/apache/hadoop/yarn/logaggregation/TestContainerLogsUtils.java

```diff
@@ -75,13 +73,12 @@ public static void createContainerLogFileInRemoteFS(Configuration conf,
     if (fs.exists(rootLogDirPath)) {
       fs.delete(rootLogDirPath, true);
     }
-    assertTrue(fs.mkdirs(rootLogDirPath));
+    fs.mkdirs(rootLogDirPath);
     Path appLogsDir = new Path(rootLogDirPath, appId.toString());
     if (fs.exists(appLogsDir)) {
       fs.delete(appLogsDir, true);
     }
-    assertTrue(fs.mkdirs(appLogsDir));
-
+    fs.mkdirs(appLogsDir);
```

Comment:
this does cut the check of the outcome...are we all happy with this?

Reply:
Thanks a lot for asking, I think we should keep this `assertTrue`.

We are currently experiencing some difficulties.

`TestContainerLogsUtils.java` uses assertTrue of `org.junit.jupiter`, `hadoop-mapreduce-client-hs` and `hadoop-mapreduce-client-app` use `TestContainerLogsUtils.java`, `hadoop-mapreduce-client-hs` and `hadoop-mapreduce-client-app` These two modules have not upgraded the junit test, so when running the test, the classes cannot be found.

`hadoop-mapreduce-client-hs` and `hadoop-mapreduce-client-app` may take some time to upgrade junit test.

aajisaka's idea can quickly fix this problem without affecting the compilation of other modules.

I will continue to follow up the `mapreduce` junit upgrade. If the mapreduce module junit upgrade is completed, I will change this back.

Reply:
@aajisaka I see your latest changes, your changes are reasonable. LGTM.

Reply:
> I thought we are happy because the directory has been removed in the previous line, but rethinking this, it's possible to fail removing the directory and then mkdir fails. Adding back the check in the next commit.

Thanks for adding it back. It makes sense. LGTM

Reply:
Thanks @slfan1989 
>hadoop-mapreduce-client-hs and hadoop-mapreduce-client-app may take some time to upgrade junit test.

I have already raised PR for both the modules. Can see some failures in `hadoop-mapreduce-client-app` - will fix it as well in that PR.

Reply:
how about removing the asserts and just calling getFileStatus(dir) after.

remember also that delete() returning false means "dir wasn't there"...not a reason to fail

Reply:
Added `getFileStatus(dir)` to check the dir exists in the latest commit. Thank you @steveloughran